src/MakeListingDisplay/components/InputField.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+/** Functional Component that creates an input field. Takes in a fieldName and fieldType as props
+to fill in the name and type attributes of the input element, respectively*/
+function InputField(props) {
+    const defaultValue = (props.fieldType === "number") ? "0" : null;
```

Comment:
I would prefer not to set null for other types. You can may be have function to get defaults based on type ?

Reply:
Created a function that returns a specific defaultValue depending on the input type: number -> "0", text -> "None" , date -> "yyyy-dd-mm" (this is defaulted to today's date), and if it's none of those, it returns "N/A". It should never return that though